### PR TITLE
Use custom fonts and enable font ligatures in code editor

### LINKED ISSUE #324: Adjust font in code editor
The code editor on GitHub seems to introduce its own styles and overwrites `font` for `<pre>` tags. This makes our `font-family` styling disappear. From short testing in the dev tools it seems that we only need to include the following style in our CSS to make our fonts work in the editor as well:

``` css
pre.CodeMirror-line {
  font: inherit;
}
```

Note that I have only tested this in the dev tools so far and it might not work if our stylesheet is loaded before the editor stylesheet.

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves #324

### CODE REVIEW

File: extension/custom.css [hunk 1]
```diff
@@ -27,3 +28,8 @@ tt,
 .wiki-wrapper .wiki-history .commit-meta code {
 	font-family: 'Fira Code', 'Ubuntu Mono', 'Droid Sans Mono', 'Operator Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
 }
+
+.CodeMirror-lines pre.CodeMirror-line {
+	-webkit-font-variant-ligatures: normal;
```
Comment (hkdobrev) [inline extension/custom.css]: I don't think the `-webkit-` prefix is needed at all. This is a Chrome extension and it ought to work on Chrome without the prefix. The "Can I use" website doesn't list any browser supporting the prefixed property - http://caniuse.com/#feat=kerning-pairs-ligatures

Comment (Turbo87) [inline extension/custom.css]: Good point. I'll change that.

Comment (Turbo87) [inline extension/custom.css]: done!

**Comment by Turbo87** [comment]
the test failures seem unrelated to this PR

**Comment by AdrianKoshka** [comment]
Just my personal opinion, but do ligatures really belong in the code editor? If I understand the title that's what this PR would do right? I'm no typography expert though : P

**Comment by Turbo87** [comment]
@AdrianKoshka without ligatures https://github.com/tonsky/FiraCode won't work properly and that was the whole point of #295

**Comment by AdrianKoshka** [comment]
Ah I see, thanks for clearing that up.Sent from my Huawei Mobile-------- Original Message --------Subject: Re: [sindresorhus/refined-github] Use custom fonts and enable font ligatures in code editor (#325)From: Tobias Bieniek To: sindresorhus/refined-github CC: Theodore Seán Tubbs ,Mention @AdrianKoshka without ligatures https://github.com/tonsky/FiraCode won't work properly and that was the whole point of #295 

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/sindresorhus/refined-github","title":"sindresorhus/refined-github","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/sindresorhus/refined-github"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Turbo87 in #325: @AdrianKoshka without ligatures https://github.com/tonsky/FiraCode won't work properly and that was the whole point of #295 "}],"action":{"name":"View Pull Request","url":"https://github.com/sindresorhus/refined-github/pull/325#issuecomment-257349880"}}}

**Pull request merged**